src/main/java/com/minecolonies/api/util/InventoryUtils.java

```diff
@@ -6,8 +6,9 @@
 import com.minecolonies.api.colony.buildings.IBuilding;
```

Comment:
Please undo this file, again this file contains a bunch of formatting changes and reorders a load of methods, making the diff unnecessarily difficult to read.
Previous time I didn't make a big deal of it, told you, but now it happened again, so it's not ok anymore.

Reply:
Cleaned it up. There's not formatting changes anymore. Just code changes.